src/jobs/run_codacy_plugins_test.yml

```diff
@@ -0,0 +1,36 @@
+description: "Runs codacy-plugins-test on a tool"
+
+executor: machine
+
+parameters:
+  codacy_plugins_test_version:
```

Comment:
I guess `version` is a better name, instead of `codacy_plugins_test_version`. Because on the config you write something like:

```yml
codacy/run_codacy_plugins_test:
  version: "1.0.2"
```

Reply:
Done